.github/workflows/ci.yml

```diff
@@ -34,6 +36,11 @@ jobs:
       run: make test
     - name: Check tidy
       run: make check-tidy
+    - name: Upload coverage report
+      uses: codecov/codecov-action@v1
```

Comment:
It looks like codecov-action@v1 includes version 1.5. It includes codecov/codecov-action#299 so it no longer requests the bash script from codecov.io. 👍